docs/resources/network_peering.md

```diff
@@ -397,6 +397,12 @@ Network Peering Connections can be imported using project ID and network peering
 $ terraform import mongodbatlas_network_peering.my_peering 1112222b3bf99403840e8934-5cbf563d87d9d67253be590a-AWS
 ```
 
+To obtain `project_id` and `peering_id` can be obtained using atlas cli. Attention: `atlas networking peering list` needs the `--provider` parameter for AZURE and GCP as it returns only `AWS` peerings by default. 
```

Comment:
replace "atlas cli" with "MongoDB Atlas CLI[https://www.mongodb.com/docs/atlas/cli/current/command/atlas/]"

Reply:
also would leverage language we already use in CLI Docs for consistency after input: Name of your cloud service provider. Valid values are AWS, AZURE, or GCP. This value defaults to "AWS".

Reply:
Changed it in https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2488